### PR TITLE
feat: export only support

### PR DESCRIPTION
**Pull request by Icemap**
refer to https://github.com/pingcap-inc/tidb2dw/issues/68

### CODE REVIEW

File: cmd/s3.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+					apiservice.GlobalInstance.APIInfo.SetServiceStatusIdle()
+				}
+			})
+		},
+	}
+
+	cmd.PersistentFlags().BoolP("help", "", false, "help for this command")
+	cmd.Flags().Var(enumflag.New(&mode, "mode", RunModeIds, enumflag.EnumCaseInsensitive), "mode", "replication mode: full, snapshot-only, incremental-only, cloud")
+	cmd.Flags().StringVar(&apiListenHost, "api.host", "0.0.0.0", "API service listen host, only available in --mode=cloud")
+	cmd.Flags().IntVar(&apiListenPort, "api.port", 8185, "API service listen port, only available in --mode=cloud")
```
Comment (Lloyd-Pottiger) [inline cmd/s3.go:95]: seems no need?

File: docs/gcs.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Google Cloud Storage
+
+## Export
+
+To export snapshot and incremental data of a TiDB Table to Google Cloud Storage:
+
+```shell
+./tidb2dw bigquery \
```
Comment (Lloyd-Pottiger) [inline docs/gcs.md:8]:
```suggestion
./tidb2dw gcs \
```

File: main.go [hunk 3]
```diff
@@ -35,10 +35,16 @@ func init() {
...
+		// replications
 		cmd.NewSnowflakeCmd(),
 		cmd.NewRedshiftCmd(),
 		cmd.NewBigQueryCmd(),
 		cmd.NewDatabricksCmd(),
+
+		// exporters
+		cmd.NewS3Cmd(),
+		cmd.NewGCSCmd(),
+		cmd.NewDatabricksCmd(),
```
Comment (wd0517) [inline main.go:47]: Duplicate registration of `NewDatabricksCmd`

File: cmd/gcs.go [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+			cdcHost, cdcPort, cdcFlushInterval, cdcFileSize, mode)
+
+		return err
+	}
+
+	cmd := &cobra.Command{
+		Use:   "gcs",
+		Short: "Export snapshot and incremental data from TiDB to GCS",
+		Run: func(_ *cobra.Command, _ []string) {
+			runWithServer(mode == RunModeCloud, fmt.Sprintf("%s:%d", apiListenHost, apiListenPort), func() {
```
Comment (wd0517) [inline cmd/gcs.go:66]: For `export only`, it should not have `cloud` mode?

Comment (Icemap) [reply cmd/gcs.go:66]: You're right, it doesn't have some long-run progress.

File: cmd/s3.go (hunk 1 above)
Comment (Icemap) [reply cmd/s3.go:95]: Deleted

**Review by Lloyd-Pottiger — APPROVED** [review]

**Pull request merged**